x-pack/solutions/security/plugins/security_solution/docs/testing/test_plans/detection_response/prebuilt_rules/prebuilt_rule_customization.md

```diff
@@ -428,4 +523,142 @@ And additionally this request contains one or more custom rules
+Then the revert rule button should be disabled
+And have an informational tooltip on hover
+```
+
+#### **Scenario: Revert prebuilt rule button shouldn't appear if rule is non-customzied**
+
+**Automation**: 1 cypress test.
+
+```Gherkin
+Given a space with at least one prebuilt rule
+And that rule is non-customized
+When a user navigates to that rule's details page
+And clicks the overflow actions button
+Then the revert rule button should not be displayed as an option
+```
+
+#### **Scenario: Revert prebuilt rule endpoint returns error if rule's base version is missing**
+
+**Automation**: 1 integration test.
+
+```Gherkin
+Given a space with at least one prebuilt rule
+And that rule is customized
+And that rule does not have an existing base version
+When a user reverts the rule
+Then the API should return a 500 error
+And the rule should remain the same
+```
+
+#### **Scenario: Revert prebuilt rule endpoint returns error if request `revision` field doesn't match the rule associated with `rule_id`**
+
+**Automation**: 1 integration test.
+
+```Gherkin
+Given a space with at least one prebuilt rule
+And that rule is customized
+And that rule has an existing base version
+When a user reverts the rule with an outdated revision field
+Then the API should return a 500 error
+And the rule should remain the same
+```
+
+#### **Scenario: Revert prebuilt rule endpoint returns error if request `version` field doesn't match the rule associated with `rule_id`**
+
+**Automation**: 1 integration test.
+
+```Gherkin
+Given a space with at least one prebuilt rule
+And that rule is customized
+And that rule has an existing base version
+When a user reverts the rule with an outdated version field
+Then the API should return a 500 error
+And the rule should remain the same
+```
+
+#### **Scenario: Revert prebuilt rule endpoint does not modify non-customized rule**
+
+**Automation**: 1 integration test.
+
+```Gherkin
+Given a space with at least one prebuilt rule
+And that rule is non-customized
+And that rule has an existing base version
+When a user reverts the rule
+Then the API should return successfully
+And the rule should remain the same
+```
+
+#### **Scenario: Revert prebuilt rule endpoint returns error if rule isn't prebuilt**
+
+**Automation**: 1 integration test.
+
+```Gherkin
+Given a space with at least custom rule
+When a user calls the revert endpoint on this rule
+Then the API should return a 500 error
+And the rule should remain the same
+```
+
+#### **Scenario: Reverting a prebuilt rule doesn't modify customization adjacent fields**
+
+**Automation**: one integration test per field.
+
+```Gherkin
+Given a space with at least one prebuilt rule
+And that rule is customized
+And that rule has an existing base version
+And that rule has a custom <field_name> field different from the base version 
+When a user reverts the rule
+Then the <field_name> field is not modified
+And the rule's `is_customized` value should be false
```

Comment:
nit: I'd move this line above the line above. The logic is the prebuilt rule's customization is reverted but customization adjacent fields aren't touched.